routes/valueRouter.js

```diff
@@ -6,13 +6,35 @@ const valueRouter = express.Router();
 // GET all items
 valueRouter.get('/', async (req, res) => {
   try {
-    const allItems = await db.query(`SELECT * FROM fair_market_value;`);
+    const { itemsLimit, pageNum } = req.query;
+
+    const allItems = await db.query(
+      `
+      SELECT *
+      FROM fair_market_value
+      ORDER BY item_id
+      ${itemsLimit ? ` LIMIT ${itemsLimit}` : ''}
+      ${pageNum ? ` OFFSET ${(pageNum - 1) * itemsLimit}` : ''};`,
+      { itemsLimit, pageNum },
+    );
     res.status(200).send(allItems);
   } catch (err) {
     res.status(500).send(err.message);
   }
 });
```

Comment:
rather than doing your pagination here, do it on the existing filter route, which was already being used for this donation items table